lib/tasks/config.js

```diff
@@ -105,6 +105,9 @@ class TaskConfig extends Task {
       cs.speechSynthesisVendor = this.synthesizer.vendor !== 'default'
         ? this.synthesizer.vendor
         : cs.speechSynthesisVendor;
+      cs.speechSynthesisLabel = this.synthesizer.label !== 'default'
```

Comment:
why would you compare label to 'default' ?  That doesn't make sense, there is no default label

Reply:
I am having same question like you, for example vendor and language are having same behavior. why is it like that?

I think for config verb, when a parameter = default means, the configure verb will use the value provided by application and not overide the parameter value in that call.

Reply:
the reason for vendor and language is that after setting applications defaults, someone might want to change something like hints with config or say but not vendor or language.  I suppose that you are right....the same could be true of label.  So I will close this